src/westpa/analysis/trajectories.py

```diff
@@ -255,6 +256,48 @@ def fget(walker, include_initpoint=True, atom_indices=None, sim_root='.'):
         self.segment_collector.show_progress = True
 
 
+class HDF5MDTrajectory(Trajectory):
+    """A MD trajectory stored by the HDF5 framework.
+
+    Parameters
+    ----------
+    atom_indices : list, default `None`
+
+    """
+
+    def __init__(self, atom_indices=None):
```

Comment:
Remove `atom_indices` parameter from constructor, and change `atom_indices=atom_indices` to `atom_indices=None` in definition of `fget()` function.